{"name":"Create: Failure","fullName":"TestPublicationPgRepoSuiteRunner/PublicationPgRepoSuit/TestPublicationPgRepo_CreateFailure","status":"passed","statusDetails":{"message":"","trace":""},"start":1727679695313,"stop":1727679695314,"uuid":"d4c0a8da-7ef9-11ef-8503-be2bab8c44a1","historyId":"a9255b9a51cb98bd12ebd8d3d567e445","testCaseId":"7dfa97f5c4965f7c962951a14277a31b","labels":[{"name":"language","value":"go1.23.1"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"MacBook-Pro-Pavel-2.local"},{"name":"thread","value":"TestPublicationPgRepoSuiteRunner/PublicationPgRepoSuit/TestPublicationPgRepo_CreateFailure"},{"name":"suite","value":"PublicationPgRepoSuit"},{"name":"package","value":"cookdroogers/internal/repo/pg"},{"name":"tag","value":"PublicationPgRepo"}],"steps":[{"name":"Failure","status":"passed","start":1727679695314,"stop":1727679695314,"steps":[{"name":"ASSERT: Error Is","status":"passed","start":1727679695314,"stop":1727679695314,"parameters":[{"name":"Error","value":"sql: connection is already closed: error in postgres repo"},{"name":"Target","value":"error in postgres repo"}]}]}]}